master/buildbot/worker/docker.py

```diff
@@ -212,9 +231,16 @@ def renderWorkerProps(self, build):
     def start_instance(self, build):
         if self.instance is not None:
             raise ValueError('instance active')
-        image, dockerfile, volumes = yield self.renderWorkerPropsOnStart(build)
```

Comment:
`renderWorkerProps` needs to be updated with the new properties. The API is this way because we want to store properties that influence the latent worker so that we can compare them and not start builds on an incompatible worker.

Reply:
Thanks!